README.md

```diff
@@ -28,6 +28,10 @@ Once you started stack you can use the following services:
 
 To check service availability you can get its status using `/status` endpoint, e.g.: http://localhost/api/file/status
 
+To upload file you can use http://localhost/api/file endpoint
```

Comment:
Let's start gathering API documentation in Wiki, here is the starting point:
https://github.com/veraPDF/verapdf-webapp-server/wiki/API-Reference